companion/discord_bot/commands.py

```
@@ -0,0 +1,442 @@
+                    else:
+                        logger.error(f"Gateway API error: {response.status} - {await response.text()}")
+                        return None
+            else:
+                async with self.session.get(url, timeout=timeout) as response:
+                    if response.status == 200:
+                        return await response.json()
+                    else:
+                        logger.error(f"Gateway API error: {response.status} - {await response.text()}")
+                        return None
+        except asyncio.TimeoutError:
+            logger.error(f"Gateway API timeout for URL: {url}")
```

Ruff (0.13.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)